### ISSUE TITLE
"Not enough storage is available to process this command" on 1st login

### ISSUE THREAD

**Issue by joaocc** [mwrock/packer-templates#4]
Hi,
Booting the vagrant box on VirtualBox 4.3.30 on Mac OSX, I get the an error message when trying to login as user vagrant

"Not enough storage is available to process this command" on 1st login.

Any idea?
Thanks

**Comment by joaocc** [comment]
This was tested with 2048MB RAM, and also happens when trying to login with the Administrator account

**Comment by joaocc** [comment]
This must have to do with some customisation I am making to Vagrantfile, since everything runs with the default one (from vagrant init...). Closing the issue. Thx and keep up the excellent work!

**Comment by mwrock** [comment]
Have not seen that before. I have a few questions. Where is the error displaying? On the guest GUI? Do you have plenty of disk space? This occurs on a vagrant box you generated from the template without any changes?

**Comment by mwrock** [comment]
Oh l see your last message. Thanks!

**Comment by joaocc** [comment]
(It is on the GUI). The physical (host) disk has plenty of space. I am cutting customizations from my Vagrantfile to try to find the cause. I will update if I find it. Thx

**Comment by joaocc** [comment]
One of the customisations I do is
```
    vbx.name = "#{my_vm_name}-vbox"
    vbx.customize [
        'modifyvm', vmid,
        '--memory', hwDefs['ram'],
        '--cpus', hwDefs['cpu'],
        '--audio', 'none',
        '--chipset', 'ich9',
    ]
```

If I don't change chipset to ICH9, everything runs ok.
If I change it, I get the "Not enough storage is available to process the command" message.

**Comment by joaocc** [comment]
If I try a wrong password, I get a "The password is incorrect. Try again".

I have already removed most of the customizations, and it still hangs. I noticed that vagrant gets stuck in sharing folders:
``´
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'mwrock/Windows2012R2' is up to date...
==> default: Setting the name of the VM: __vag_vbxwin_default_1437864847654_19642
==> default: Clearing any previously set forwarded ports...
==> default: Fixed port collision for 5985 => 55985. Now on port 2200.
==> default: Fixed port collision for 22 => 2222. Now on port 2201.
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 5985 => 2200 (adapter 1)
    default: 22 => 2201 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
==> default: Machine booted and ready!
Sorry, don't know how to check guest version of Virtualbox Guest Additions on this platform. Stopping installation.
==> default: Checking for guest additions in VM...
    default: No guest additions were detected on the base box for this VM! Guest
    default: additions are required for forwarded ports, shared folders, host only
    default: networking, and more. If SSH fails on this machine, please install
    default: the guest additions and repackage the box to continue.
    default: 
    default: This is not an error message; everything may continue to work properly,
    default: in which case you may ignore this message.
==> default: Setting hostname...
==> default: Mounting shared folders...
    default: /vagrant => /vms/test/__vag_vbx.win
´´´
And it hangs there indefinitely.
If I try to stop it from another terminal windows (even "vagrant halt -f") I get
```
An action 'halt' was attempted on the machine 'default',
but another process is already executing an action on the machine.
Vagrant locks each machine for access by only one process at a time.
Please wait until the other Vagrant process finishes modifying this
machine, then try again.

If you believe this message is in error, please check the process
listing for any "ruby" or "vagrant" processes and kill them. Then
try again.
```

Occasionally, the VM boots into blue screens of death and keeps on booting and crashing...
´´´
Your PC ran into a problem and needs to restart. We're just collecting some error info, and the we'll restart for you.
If you'd like to know more, you can search online later for this error: CRITICAL_PROCESS_DIED
´´´

**Comment by joaocc** [comment]
Oddly enough, things stop working once i enable this:
´´´
    config.vm.hostname = "#{my_vm_name}-vm"
´´´
Hope this helps

**Comment by mwrock** [comment]
For the shared folder, I probably should add guest additions to the builder. I removed it because it would intermittently cause packer to fail during the upload.

Regarding the computername, I have seen vagrant blue screen changing the name on windows VMs. Thought that was fixed in 1.7.3.

**Comment by mwrock** [comment]
I'm closing this but have a couple notes to add:

1. The guest additions are being added in a commit today.
2. I have saw alot of  blue screens with "CRITICAL_PROCESS_DIED" when working on Windows nano and perhaps its related to what you saw. The blue screens would occur when cloning a good VM (which the virtualbox builder does) if it had not been sysprepped during shutdown in the `shutdown_command`. The fix is either to sysprep or to ensure that the `hardwareuuid` of both the original vm and the cloned vm are identical. See http://www.hurryupandwait.io/blog/a-packer-template-for-windows-nano-server-weighing-300mb for details.
